types/sortable-options.d.ts

```diff
@@ -0,0 +1,204 @@
+import { Sortable } from './sortable';
+import { SortableEvent, SortableMoveEvent } from './sortable-event';
+
+export interface SortableGroupOptions {
+	/**
+	 * group name
+	 */
+	name: string;
+	/**
+	 * ability to move from the list. clone — copy the item, rather than move.
+	 */
+	pull?: boolean | 'clone' | ((to: Sortable, from: Sortable) => boolean | string);
+	/**
+	 * whether elements can be added from other lists, or an array of group names from which elements can be taken.
+	 */
+	put?: boolean | string | ReadonlyArray<string> | ((to: Sortable) => boolean);
+	/**
+	 * a canonical version of pull, created by Sortable
+	 */
+	checkPull?: ((to: Sortable, from: Sortable) => boolean | string);
+	/**
+	 * a canonical version of put, created by Sortable
+	 */
+	checkPut?: (to: Sortable) => boolean;
+	/**
+	 * revert cloned element to initial position after moving to another list.
+	 */
+	revertClone?: boolean;
+}
+
+export interface SortableOptions {
```

Comment:
This is a breaking change. It should be `Options`, not `SortableOptions`.

I'd also prefer to have plugins `Options` separate from the normal plugins.

I may issue a pull on your repo with this change as i have all the types done.

```ts
export interface Options extends SortableOptions, OnSpillOptions, AutoScrollOptions, MultiDragOptions {}
```